src/site/twitch.tv/modules/chat/ChatList.vue

```diff
@@ -451,6 +452,25 @@ watch([alt, isHovering], ([isAlt, isHover]) => {
 	}
 });
 
+// Assign highlight to your own message
+watch(
+	[identity, showSelfHighlights],
+	([identity, enabled]) => {
+		if (enabled) {
+			chatHighlights.define("~self", {
+				test: (msg) => msg.author?.id === identity?.id,
```

Comment:
This condition has the possibility to become `undefined === undefined`, which would return `true`.

Suggest adding an additional checks

```suggestion
		if (enabled && identity) {
			chatHighlights.define("~self", {
				test: (msg) => !!(msg.author && identity) && msg.author.id === identity.id,
```